source/back.tex

```diff
@@ -77,15 +77,19 @@
   ACM Trans.\ Math.\ Soft., Vol.\ 16, No.\ 1, pp.\ 1--17, Mar.\ 1990.
 \bibitem{lapack}
   E.\ Anderson, Z.\ Bai, C.\ Bischof, S.\ Blackford, J.\ Demmel, J.\ Dongarra,
-  J.\ Du Croz, A.\ Greenbaum, S.\ Hammarling, A.\ McKenney, D.\ Sorensen
+  J.\ Du Croz, A.\ Greenbaum, S.\ Hammarling, A.\ McKenney, and D.\ Sorensen,
   \doccite{LAPACK Users' Guide, Third Edition}.
   SIAM, Philadelphia, PA, USA, 1999.
 \bibitem{blas-std}
-  L. Susan Blackford, Ames Demmel, Jack Dongarra, Iain Duff, Sven Hammarling,
+  L.\ Susan Blackford, James Demmel, Jack Dongarra, Iain Duff, Sven Hammarling,
   Greg Henry, Michael Heroux, Linda Kaufman, Andrew Lumbsdaine, Antoine Petitet,
-  Roldan Pozo, Karin Remington, R. Client Whaley
+  Roldan Pozo, Karin Remington, and R.\ Client Whaley,
   \doccite{An Updated Set of Basic Linear Algebra Subprograms (BLAS)}.
```

Comment:
Oh god, could you send those existing fixes as a separate PR please? That's horrendous, well spotted!

Reply:
Thanks! : - )  I'll move those fixes into a separate PR.

Reply:
I've removed these fixes from this PR, and moved them to PR https://github.com/cplusplus/draft/pull/7507 .  Thanks! : - )